volumes/miovision/api/intersection_tmc.py

```diff
@@ -370,8 +370,9 @@ def pull_data(conn, start_time, end_time, intersection, path, pull, key, dupes):
             table.extend(table_veh)
             table.extend(table_ped)
 
-        logger.info('Completed data pulling from {0:s} to {1:s}'
-                    .format(c_start_t, c_end_t))
+        logger.info('Completed data pulling from %s to %s'
```

Comment:
FYI you can append variable to be formatted into the string without calling a formatting function
https://docs.python.org/3/howto/logging.html#logging-variable-data